libs/algo/odc/algo/io.py

```diff
@@ -52,6 +56,32 @@ def compute_native_load_geobox(
     )
 
 
+def choose_transform_path(
+    src_crs: str, dst_crs: str, transform_code: Optional[str] = None
+) -> str:
+    # generally it is a bad idea to make aoi global
+    # here is to accommodate the fact that 326xx not overlapping with 3577
+    area_of_interest = aoi.AreaOfInterest(-180.0, -90.0, 180.0, 90)
```

Comment:
Maybe you should be able to pass in an area of interest?  It would make sense (for us) to be able to restrict to Australia-ish for example, but obviously Africa (and Antarctica) would need to be able to specify their own AOIs.

In an ideal world where people used CRSs properly we could default AOI to the valid region of the destination CRS.

Reply:
agreed. fixed in the commit below.